json_parsers/json_parser2.py

```diff
@@ -1,36 +1,30 @@
-import json
-
-path="D:\\python\\input.json"
-
-f=open(path,'r')
-
-data=f.read()
-
-f.close()
-
-x=json.loads(data)
-
-print("id1:",x["data"][0]["_id"])
-print("id2:",x["data"][1]["_id"])
-
-
-print("*"*60)
 
 import json
 
-path="D:\\python\\input2.json"
-
-
-f=open(path,'r')
-
-data=f.read()
-
-f.close()
-
-x=json.loads(data)
-
-print(x["mongodb"])
-
-print(type(x))
-
-
+class JSON: 
+    
+    def json_read(self,path):
+    
```

Comment:
you can split this function 

def read_file(self, filepath):
      pass

def load_json_data(self, json_text):
      pass